src/components/dialogs/parameters/map-parameters.js

```diff
@@ -57,11 +33,11 @@ export const MapParameters = ({ hideParameters }) => {
         },
     ];
 
-    useEffect(() => {
-        setDisabledFlowAlertThreshold(
-            lineFlowColorModeLocal === 'nominalVoltage'
-        );
-    }, [lineFlowColorModeLocal]);
+    const [lineFlowColorModeLocal] = useParameterState(
```

Comment:
you shouldn't create a local state param if you don't change it. It's always the value of the global state (and lagging after one rerender)